shared/haml/user_header.haml

```diff
@@ -46,10 +46,11 @@
       - else
         %i.fa.fa-user
       .user_options{style: 'display: none'}
+        - short_name = ERB::Util.h(current_user.short_name)
+        .display_name{'data-id': current_user.id, 'data-shortname': short_name}= I18n.t("#{loc_prefix}display_name", username: short_name)
         - if current_user.can_pair?
```

Comment:
This doesn't work the way that I'd hoped on Pegasus (I added a new string to the i18n gsheet, checked that it synced, committed the change on staging, then rebased this branch onto staging). Instead of the correct display name, I'm just seeing a generic "Student" display name.
<img width="257" alt="screen shot 2018-09-24 at 8 02 32 pm" src="https://user-images.githubusercontent.com/12300669/45990642-2c5f2500-c036-11e8-9f68-1060b32f17d7.png">
If things go sideways, I'd expect to see the "translation-missing..." string. Any ideas?

Reply:
This is a very good point. It somehow never occurred to me that I could avoid making this a i18n string. Ay yi yi! Thank you, @Hamms

Reply:
I don't think the issue actually has anything to do with i18n.  For some reason, I'm hitting   the default `short_name: I18n.t("nav.user.#{user_type}")` from https://github.com/code-dot-org/code-dot-org/blob/b65f5bedacc0eb1ea4d899c667b48fb0fd7b35e2/dashboard/app/views/layouts/_header.html.haml#L49

@wjordan Is there something I'm missing to access current_user from Pegasus? Similar to how we did for the "Hi, <username>" button. Seems potential related to the work here: https://github.com/code-dot-org/code-dot-org/pull/16614